admin/scripts/modules/reporty/reporty.php

```diff
@@ -31,7 +31,8 @@
   ['maily-prihlaseni',  'Maily – přihlášení na GC (vč. unsubscribed)', ['csv', 'html']],
   ['maily-neprihlaseni','Maily – nepřihlášení na GC',             ['csv', 'html']],
   ['maily-vypraveci',   'Maily – vypravěči (vč. unsubscribed)',   ['csv', 'html']],
-  ['maily-vsichni',     'Maily – všichni',                        ['csv', 'html']],
+  ['maily-dle-data-ucasti?start=0',     'Maily - nedávní účastníci (prvních 2000)'],
```

Comment:
Zarovnat pls. Pokud to nejde zarovnat s ostatnima nejak rozumne, tak jen jednu mezeru za carkou a nezarovnavat.